### DISCUSSION TITLE
Reduce the amount of steps/click

### DISCUSSION THREAD

**Discussion by TJJP** [Franciscan-CS-Club/fix-franciscan-ordering#17] [Ideas]
Right now you have to make a lot of clicks and it is a hassle to order because it takes too long, but if it was simplified then it would be so much easier to order!

**Comment by 2br-2b** [comment]
I think this is a must, but how would you suggest simplifying this? Maybe save your most recent order of a burger (toppings, etc.) and autofill that next time you make a burger?

**Reply by TJJP** [reply]
I think a saving feature would be great! I think the easiest way would be to have a cart/review order page and then a place order page. Where it is all in one step, so you can select payment and name. Like Amazon has a fast checkout option, that makes it faster to check out by remembering your card number. 

Currently, you have to Confirm selection, select at the time, and then Confirm selection AGAIN, then check out by selecting a payment option.
